processing/src/main/java/io/druid/segment/BitmapOffset.java

```diff
@@ -51,8 +52,9 @@ private IntIterator newIterator()
     if (!descending) {
       return bitmapIndex.iterator();
     }
-    // ImmutableRoaringReverseIntIterator is not cloneable.. looks like a bug
-    // update : it's fixed in 0.5.13
+    if (bitmapIndex instanceof WrappedImmutableRoaringBitmap) {
+      return ((WrappedImmutableRoaringBitmap) bitmapIndex).getBitmap().getReverseIntIterator();
+    }
     int i = bitmapIndex.size();
     int[] back = new int[bitmapIndex.size()];
```

Comment:
we noted this before as well ( https://github.com/druid-io/druid/pull/2014/files#r48668522 ).